coupler/atmos_ocean_fluxes.F90

```diff
@@ -424,7 +424,7 @@ end function aof_set_coupler_flux
   !! @throw FATAL, "Num_flags is negative for [name]: [num_flags]"
   !! @throw FATAL, "Problem dumping fluxes tracer tree"
   !! @throw FATAL, "Number of fluxes does not match across the processors: [gas_fluxes%num_bcs] fluxes"
-  subroutine atmos_ocean_fluxes_init(gas_fluxes, gas_fields_atm, gas_fields_ice, verbosity)
+  subroutine atmos_ocean_fluxes_init(gas_fluxes, gas_fields_atm, gas_fields_ice, verbosity, use_r4_kind)
```

Comment:
do you think there will be a case where the user wants these three fluxes to be set at different precisions?

Reply:
I do not know honestly. It could be added using similar logic later on if we need it.